Lib/email/test/test_email.py

```diff
@@ -920,6 +920,22 @@ def test_long_lines_with_different_header(self):
 
 """)
 
+    def test_oversized_header_input(self):
+        mail = email.message_from_string("""From: <postmaster@example.com>
+To: <bounce@example.com>
+Subject: demo
+X-Overlong-Header-Name-causes-python-mail-to-crash-in-re-serialization-example: 
```

Comment:
There is an invisible trailing space at the end of the line. Make it visible. E.g. `\x20`.